pkgs/development/web/playwright-test/default.nix

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
We should stop using node2nix if possible

Reply:
Noted.